### PR TITLE
Add redirects for broken links from Google search

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Google search results can still directs people to the former location of our Upsun GS guides, so we need redirects.

### CODE REVIEW

File: .platform/routes.yaml [hunk 1]
```diff
@@ -364,6 +364,13 @@ https://docs.upsun.com/:
       "/anchors/environments/cancel-activity.html": { "to": "/environments/cancel-activity.html", "code": 301, "prefix": false }
       "/anchors/admin/servers.html": { "to": "/administration/servers.html", "code": 301, "prefix": false }
 
+      # Redirects for moved GS guides
```
Comment (AnouckColson) [inline .platform/routes.yaml:367]:
```suggestion
      # Redirects for moved Upsun GS guides
```

File: .platform/routes.yaml [hunk 2]
```diff
@@ -364,6 +364,13 @@ https://docs.upsun.com/:
       "/anchors/environments/cancel-activity.html": { "to": "/environments/cancel-activity.html", "code": 301, "prefix": false }
       "/anchors/admin/servers.html": { "to": "/administration/servers.html", "code": 301, "prefix": false }
 
+      # Redirects for moved Upsun GS guides
+      "/get-started/express/(.*).html": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
+      "/get-started/flask/(.*).html": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
```
Comment (sivachandran) [inline .platform/routes.yaml:369]: I got 404 for https://docs.upsun.com/get-started/flask/deploy/prepare.html. Shouldn't this be `/get-started/flask/deploy/(.*.html)` ?

I also tested the Upsun docs page from the CI. It is not redirecting.

Comment (AnouckColson) [reply .platform/routes.yaml]: The former Get Started > Flask section had multiple subpages that may also still pop up as Google search results, so we want to tackle all those with a single redirect. That being said, my proposal was indeed wrong, I think maybe `/get-started/flask(.*)` would be better suited to address the entire section in a simple way? @chadwcarlson for review please.

File: .platform/routes.yaml [hunk 3]
```diff
@@ -364,6 +364,13 @@ https://docs.upsun.com/:
       "/anchors/environments/cancel-activity.html": { "to": "/environments/cancel-activity.html", "code": 301, "prefix": false }
       "/anchors/admin/servers.html": { "to": "/administration/servers.html", "code": 301, "prefix": false }
 
+      # Redirects for moved Upsun GS guides
+      "/get-started/express(.*)": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
+      "/get-started/flask(.*)": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
+      "/get-started/laravel(.*)": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
+      "/get-started/nextjs(.*)": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
+      "/get-started/strapi(.*)": { "to": "/get-started/stacks.html", "code": 301, "prefix": false }
```
Comment (chadwcarlson) [inline .platform/routes.yaml:372]:
```suggestion
      "/get-started/express/": { "to": "/get-started/stacks/express.html", "code": 301, "prefix": true }
      "/get-started/flask/": { "to": "/get-started/stacks/flask.html", "code": 301, "prefix": true }
      "/get-started/laravel/": { "to": "/get-started/stacks/laravel.html", "code": 301, "prefix": true }
      "/get-started/nextjs/": { "to": "/get-started/stacks/nextjs.html", "code": 301, "prefix": true }
      "/get-started/strapi/": { "to": "/get-started/stacks/strapi.html", "code": 301, "prefix": true }
```

**Comment by chadwcarlson** [comment]
https://docs.upsun.com.redirects-gs-uiluhsq-ucq44jg6ofare.eu-5.platformsh.site/get-started/flask/deploy/prepare.html

File: .platform/routes.yaml [hunk 4]
```diff
@@ -364,6 +364,13 @@ https://docs.upsun.com/:
       "/anchors/environments/cancel-activity.html": { "to": "/environments/cancel-activity.html", "code": 301, "prefix": false }
       "/anchors/admin/servers.html": { "to": "/administration/servers.html", "code": 301, "prefix": false }
 
+      # Redirects for moved Upsun GS guides
+      "/get-started/express/": { "to": "/get-started/stacks/express.html", "code": 301, "prefix": true }
+      "/get-started/flask/": { "to": "/get-started/stacks/flask.html", "code": 301, "prefix": true }
+      "/get-started/laravel/": { "to": "/get-started/stacks/laravel.html", "code": 301, "prefix": true }
+      "/get-started/nextjs/": { "to": "/get-started/stacks/nextjs.html", "code": 301, "prefix": true }
+      "/get-started/strapi/": { "to": "/get-started/stacks/strapi.html", "code": 301, "prefix": true }
```
Comment (chadwcarlson) [inline .platform/routes.yaml:372]:
```suggestion
      "/get-started/express/": { "to": "/get-started/stacks/express.html", "code": 301, "prefix": true, "append_suffix": false }
      "/get-started/flask/": { "to": "/get-started/stacks/flask.html", "code": 301, "prefix": true, "append_suffix": false }
      "/get-started/laravel/": { "to": "/get-started/stacks/laravel.html", "code": 301, "prefix": true, "append_suffix": false }
      "/get-started/nextjs/": { "to": "/get-started/stacks/nextjs.html", "code": 301, "prefix": true, "append_suffix": false }
      "/get-started/strapi/": { "to": "/get-started/stacks/strapi.html", "code": 301, "prefix": true, "append_suffix": false }
```

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**